File: 백준/Bronze/10818. 최소， 최대/최소， 최대.c
#include <stdio.h>

int main(void) {
    int n;
    scanf("%d", &n);
    int* arr = (int*)malloc(sizeof(int) * n);
    for (int i = 0; i < n; i++) {
        scanf("%d", &arr[i]);
    }
    int max = arr[0];
    int min = arr[0];
    for (int j = 0; j < n; j++) {
        if (max < arr[j]) {
            max = arr[j];
        }
        if (min > arr[j]) {
            min = arr[j];
        }
    }
    printf("%d ", min);
    printf("%d", max);
    
}

File: 백준/Bronze/10818. 최소， 최대/최소， 최대.py
n = int(input())
arr = list(map(int, input().split()[:n]))

min = arr[0]
max = arr[0]
for i in arr:
    if(i > max):
        max = i
    if (i < min):
        min = i
print('{} {}'.format(min, max))